setup.py

```diff
@@ -6,7 +6,7 @@
       py_modules = ['Ska.quatutil'],
       url = 'http://cxc.harvard.edu/mta/ASPECT/tool_doc/pydocs/Ska.quatutil.html',
       test_suite = 'nose.collector',
-      version='0.02',
+      version='0.03',
```

Comment:
Technically this seems more like a bug fix, which would be 0.03.1.  But whatever.